platform/ios/src/MGLMapView.h

```diff
@@ -1200,17 +1200,26 @@ MGL_EXPORT IB_DESIGNABLE
 
  Assigning a new array to this property selects only the first annotation in
  the array.
+
+ @note In versions prior to `4.0.0` if the annotation was offscreen it was not selected.
```

Comment:
This note here is good. Let’s also make explicit whether or not setting `selectedAnnotations` will move the viewport (and point them in the direction of `selectAnnotation:animated:` if they need more control over that).